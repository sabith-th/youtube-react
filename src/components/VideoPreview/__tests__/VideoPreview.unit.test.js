import { shallow } from "enzyme";
import React from "react";
import { VideoPreview } from "../VideoPreview";

describe("Video Preview", () => {
  test("renders vertically", () => {
    const wrapper = shallow(<VideoPreview />);
    expect(wrapper).toMatchSnapshot();
  });
  test("renders horizontally", () => {
    const wrapper = shallow(<VideoPreview horizontal={true} />);
    expect(wrapper).toMatchSnapshot();
  });
});
