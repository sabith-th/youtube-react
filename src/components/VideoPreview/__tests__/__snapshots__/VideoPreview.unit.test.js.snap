// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video Preview renders horizontally 1`] = `<div />`;

exports[`Video Preview renders vertically 1`] = `<div />`;
